src/main/java/mate/academy/spring/controller/OrderController.java

```diff
@@ -34,14 +35,16 @@ public OrderController(ShoppingCartService shoppingCartService,
     }
 
     @PostMapping("/complete")
-    public OrderResponseDto completeOrder(@RequestParam Long userId) {
-        ShoppingCart cart = shoppingCartService.getByUser(userService.get(userId));
+    public OrderResponseDto completeOrder(@RequestParam Authentication authentication) {
+        ShoppingCart cart = shoppingCartService
+                .getByUser(userService.findByEmail(authentication.getName()).get());
```

Comment:
```suggestion
                .getByUser(userService.findByEmail(authentication.getName()).get());
```
let's throw an exception in case user is not found, the same for other similar cases